### PR TITLE
[candi] Switch base images from Debian to Ubuntu & update BASE_UBUNTU to Jammy

### LINKED ISSUE #2679: Add automated periodic trivy tests.
### Preflight Checklist

- [X] I agree to follow the [Code of Conduct](https://github.com/deckhouse/deckhouse/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/deckhouse/deckhouse/issues) for an issue that matches the one I want to file, without success.

### Use case. Why is this important?

We need actual info about CVE's in Deckhouse to fix them ASAP.

### Proposed Solution

1. Run CI task to check all deckhouse images - `make cve-report`. This task is informational, so run should be successful in any cases.
2. Run CI task to check base images - `make cve-base-images`. This task is informational, so run should be successful in any cases. 
3. Get rid of base images by changing BASE_DEBIAN* images to BASE_UBUNTU*.

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by gmaiorov**
## Description

This PR changes Debian base images to Ubuntu base images in all modules and updates BASE_UBUNTU from Bionic to Jammy.

Linstor-related modules was update previously by @kvaps in #2561.
This PR removes separate BASE_UBUNTU_JAMMY image introduced in said PR.

Debian-based `BASE_GOLANG_*` wasn't changed because there is no Ubuntu-based `golang` images found on [Golang official Docker Hub page](https://hub.docker.com/_/golang/tags?page=1&name=bionic).

`modules/340-monitoring-kubernetes/images/ebpf-exporter` is incompatible with Ubuntu Jammy & was left on Ubuntu Bionic.
It is intended for update in separate PR, thus temporary BASE_UBUNTU_BIONIC was introduced as workaround.
. 
<details>
<summary>Changes were tested using GCP e2e and on dev cluster; all pods are up and no unexpected alerts are firing or pending.</summary>

Prometheus_alert_page

pods_p1

pods_p2

</details>

## Why do we need it, and what problem does it solve?

Together with #2763 closes #2679.
## What is the expected result?

- BASE_DEBIAN* images removed from repository;
- BASE_UBUNTU updated to Jammy;
- BASE_UBUNTU_BIONIC introduced as workaround for `ebpf_exporter`;
- Multiple d8 images switched from Debian Buster or Bullseye to Ubuntu Jammy.

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: candi
type: feature
summary: Switch base images from Debian to Ubuntu & update BASE_UBUNTU to Jammy
impact: |
  This PR will restart following components (if used):
  * ebs-csi-plugin in cloud-provider-aws module;
  * azuredisk-csi in cloud-provider-azure;
  * pd-csi-plugin in cloud-provider-gcp;
  * kube-controller-manager in control-plane-manager;
  * grafana in prometheus module;
  * ui-proxy in cilium-hubble module;
  * vector in log-shipper module;
  * cinder-provider-openstack and cloud-controller-manager in cloud-provider-openstack module;
  * vsphere-csi-plugin and vsphere-csi-plugin-legacy in cloud-provider-vsphere module;
  * operator, pilot and proxyv2 in istio module;
  * grafana-agent in flant-integration module.
impact_level: high
```

### CODE REVIEW

File: candi/image_versions.yml [hunk 1]
```diff
@@ -24,3 +24,4 @@ BASE_RUST: "rust:1.57.0@sha256:7d2592d1a6a73055fbadc0c76868046736ce5fa834ed5c201
 BASE_SCRATCH: "spotify/scratch@sha256:3ab2f8af40415f6aec66bb953119a3bdc3ec4b542c8393c9b1a40ef6cb5a4136"
 BASE_SHELL_OPERATOR: "flant/shell-operator:v1.0.12@sha256:3927077b59c89a44ca945d11613b424ae22b285c070df2b28f27b62f2f68abcb"
 BASE_UBUNTU: "ubuntu:bionic-20220531@sha256:2776b2dc16a72a7dabcbb0b0b70c181922c45e9a08aa740ddc65f53511863e32"
+BASE_UBUNTU_FOCAL: "ubuntu:focal-20221019@sha256:b029e639b47424ef2bc862e09dd7c805cc522ec604f4755ed4028c80c8042319"
```
Comment (kvaps) [inline candi/image_versions.yml:27]: Can't we bump existing `BASE_UBUNTU` version to focal?

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've started the workflow for label 'e2e/run/gcp'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've started the workflow for label 'e2e/run/gcp'...

**Review by zuzzas — APPROVED** [review]

**Review by shvgn — APPROVED** [review]
LGTM for flant-pricing module

**Review by kvaps — APPROVED** [review]
LGTM

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @kvaps. I've started the workflow for label 'e2e/run/static'...

**Review by fl64 — DISMISSED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've started the workflow for label 'e2e/run/azure'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've started the workflow for label 'e2e/run/gcp'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've started the workflow for label 'e2e/run/azure'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Review by nabokihms — APPROVED** [review]
Grafana, Vector ok. Approve.

**Comment by gmaiorov** [comment]
Rebased onto main due to fixed Azure e2e tests.

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've started the workflow for label 'e2e/run/azure'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've started the workflow for label 'e2e/run/gcp'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @gmaiorov. I've started the workflow for label 'e2e/run/vsphere'...

**Review by RomanenkoDenys — APPROVED** [review]
LGTM

**Review by shvgn — APPROVED** [review]

**Review by kvaps — APPROVED** [review]

**Review by nabokihms — APPROVED** [review]

**Review by zuzzas — APPROVED** [review]

**Review by apolovov — APPROVED** [review]

**Review by konstantin-axenov — APPROVED** [review]

**Pull request merged**